Makefile.include

```diff
@@ -178,9 +178,11 @@ export TOOLCHAIN
 export PREFIX ?= $(if $(TARGET_ARCH),$(TARGET_ARCH)-)
 
 # Add standard include directories
-INCLUDES += -I$(RIOTBASE)/core/include -I$(RIOTBASE)/drivers/include -I$(RIOTBASE)/sys/include
+INCLUDES += -I$(RIOTBASE)/core/include
+INCLUDES += -I$(RIOTBASE)/sys/include
 INCLUDES += -I$(RIOTCPU)/$(CPU)/include
 INCLUDES += -I$(RIOTBOARD)/$(BOARD)/include
+INCLUDES += -I$(RIOTBASE)/drivers/include
```

Comment:
Changing this does not have the full expected behavior.
I understood the goal is that a board can override some driver headers. But currently, it would only work if overridden in the board or the CPU itself and not in `board_common` as common includes would only be added later by `include $(RIOTBOARD)/$(BOARD)/Makefile.include`.

Fixing this is a more general problem, if a board should be able to override any header, these `INCLUDES` should not be in this order at all.